src/Core/Rexl.Harness/Storage.cs

```diff
@@ -61,12 +62,16 @@ public virtual Task<(Link full, Stream stream)> LoadStreamAsync(Link linkCtx, Li
 
     /// <summary>
     /// Storage abstraction providing default methods to read/write file streams
-    /// from/to the local file system.
+    /// from/to the local file system and http.
     /// </summary>
     public abstract class LocalFileStorage : Storage
     {
+        private readonly HttpClient _http;
+
         protected LocalFileStorage()
         {
+            _http = new HttpClient();
+            _http.DefaultRequestHeaders.UserAgent.ParseAdd("Mozilla/5.0 (compatible; Rexl/1.0)");
```

Comment:
I'm assuming this header is only needed for Mozilla integration?

Reply:
This is needed for some web servers. For example wikimedia fails if there is no user agent. The sample notebooks added both use images from wikimedia.